lib/till.rb

```diff
@@ -0,0 +1,41 @@
+class Till
+
+  attr_reader :items, :prices
+
+  def initialize
+    @items = []
+    list_prices
+  end
+
+  def add args
+    item = args[:item]
+    quantity = args[:quantity]
+    items.push({item: item, quantity: quantity})
+  end
+
+  def list_prices
+    file = File.read('itemlist.json')
+    @prices = JSON.parse(file).first['prices'].first
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.